R/install-github.r

```diff
@@ -287,3 +287,14 @@ remote_sha.github_remote <- function(remote, url = "https://github.com", ...) {
     unname(res[found[1]])
   }, error = function(e) NA)
 }
+
+download_github <- function(path, url, ...) {
+  request <- GET(url, ...)
```

Comment:
Gahhhhh!

Ok I reviewed this whole thing again, 413300a should actually be correct.